tsdb/head.go

```diff
@@ -784,6 +784,7 @@ func (h *Head) Init(minValidTime int64) error {
 	}
 	// Backfill segments from the most recent checkpoint onwards.
 	for i := startFrom; i <= endAt; i++ {
+		walSegmentDuration := time.Now()
```

Comment:
nit: as it's not really a duration I'd just go with `start` e.g.
as we do in other places https://github.com/search?q=repo%3Aprometheus%2Fprometheus%20%22%3A%3D%20time.Now()%22&type=code